README.md

```diff
@@ -1,10 +1,35 @@
 # MetaMask Code of Conduct
 
-Document created here as a starting point.
+> “Hello babies. Welcome to Earth. It's hot in the summer and cold in the winter. It's round and wet and crowded. On the outside, babies, you've got a hundred years here. There's only one rule that I know of, babies-"God damn it, you've got to be kind.”
 
-## A placeholder from Cat's Cradle
+― Kurt Vonnegut
 
-“Hello babies. Welcome to Earth. It's hot in the summer and cold in the winter. It's round and wet and crowded. On the outside, babies, you've got a hundred years here. There's only one rule that I know of, babies-"God damn it, you've got to be kind.”
+Like the technical community as a whole, the MetaMask team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
 
-― Kurt Vonnegut
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the MetaMask project or MetaMask Software Foundation. This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```

Comment:
Things to change:
- Metamask Software Foundation (not 100% sure about this one)
- DSF events

On a separate note, should we mention ConsenSys here at all?